lib/Field/index.js

```diff
@@ -8,12 +8,13 @@ import FieldValidations from './FieldValidations';
 import FieldActions from './FieldActions';
 import ExpandingTextArea from '../ExpandingTextArea';
 import EditableTextWrapper from '../EditableTextWrapper';
+import { RepeatableSettings } from './RepeatableSettings';
 
 const Field = props => {
-  const classNames = cx(`field field--${props.dir}`, {
+  const classNames = cx(`group field field--${props.dir}`, {
```

Comment:
This `group` class is used by tailwind for the group-hover, down inside `RepeatableSettings`. IMO it's a shame that this is not nearer, yet hidden in another file...